File: Scrumban.Model/Task.cs
using System;

namespace Scrumban.Model {

    public class Task {

        public Guid Id { get; set; }

        public string Status { get; set; }

        public string Name { get; set; }

        public Guid ProjectId { get; set; }

        public Guid? UserId { get; set; }

        public virtual Project Project { get; set; }

        public virtual User User { get; set; }

    }

}

File: Scrumban/ClientApp/src/store/Task.ts
export type TaskStatus = 'Zaplanowane' | 'Oczekujące' | 'W trakcie' | 'Testy' | 'Ukończone';

export interface Task {
    id: string;
    status: TaskStatus;
    name: string;
    project?: {
        id: string;
        name: string;
    }
    user?: {
        id: string;
        fullName: string;
    }
    userId?: string;
}